app/controllers/submissions_controller.rb

```diff
@@ -18,8 +18,12 @@ class SubmissionsController < ApplicationController
   def index
     @submissions = @assessment.submissions.order("created_at DESC")
 
+    # Unsure whether this is actually useless. Keeping just-in-case
+    # rubocop:disable Lint/UselessAssignment
     assign = @assessment.name.gsub(/\./, "")
     modName = (assign + @course.name.gsub(/[^A-Za-z0-9]/, "")).camelize
```

Comment:
Yes these assigns are useless - they're all unused local variables

Reply:
We only really need to worry/reason about it if the variable is preceded with a `@` or `:`, but in that case Rubocop won't touch it